server/src/util.mjs

```diff
@@ -45,6 +47,52 @@ function createTmpFile(prefix, postfix) {
   return tmpObj;
 }
 
+function fSuccess(onMethod, result, msg) {
+  logger.info(
+    `${msg}: Sent event ${onMethod} with result ${JSON.stringify(result)}`
+  );
+};
+
+function fErr(onMethod) {
+  logger.info(
+    `Could not send ${onMethod} event because no listener is active`
+  );
+};
+
+function fFatalErr() {
+  logger.info(`Internal error`);
+};
+
+function sendEvent(ws, userId, onMethod, result, msg) {
```

Comment:
I see what you're doing here (and with sendBroadcastEvent, but I don't think there should be a dependency from util.mjs to any other file; I'd like to keep it for independent "generic utilities". But you can put these functions inside events.mjs if you want, or just go back to where you're passing in the function pointers, but just pass util.fSuccess, etc.

Reply:
changes done, please check and confirm